specification/signalr/resource-manager/Microsoft.SignalRService/preview/2021-06-01-preview/signalr.json

```diff
@@ -1372,7 +1373,8 @@
           "description": "Properties of the private endpoint connection",
           "x-ms-client-flatten": true
         }
-      }
+      },
+      "required": ["properties"]
```

Comment:
What's the purpose of this PR? If you are going to apply it to lastest `api-version` too, I would prefer to update our RP source code first otherwise the change will be erased. Since the swagger files are auto-generated via source code.

Reply:
This PR is a draft, to help API scenario generation in our internal test.